frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -203,18 +204,6 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
     [spiderfierRef, map, maxZoom, spiderfyOnMaxZoom, supercluster, zoomToBoundsOnClick],
   );
 
-  /**
-   * Update the map view to center on the selected parcel when it changes.
-   */
-  useDeepCompareEffect(() => {
-    if (!!selected?.parcelDetail?.latitude && !!selected?.parcelDetail?.longitude) {
-      map.setView(
-        { lat: selected.parcelDetail.latitude, lng: selected.parcelDetail.longitude },
-        Math.max(maxZoom as number, map.getZoom()),
-      );
-    }
-  }, [selected]);
```

Comment:
The view is set whenever a pin is clicked, don't need it repeated here. Also the zoom is done when "view details" or "Update" is clicked, this would trigger the map to zoom any time a pin is clicked now

Reply:
I think this will take away the logic that jumps back to a map marker after it is saved. I'm not sure that there is actually a requirement for that however so maybe that is fine.

Reply:
I just tried, it doesn't seem to zoom back out after the sidebar closes if that's what you mean